src/api/users.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1844;
+
+export const getUsers = () => {
+  return client.get<User[]>(`/users`);
+};
```

Comment:
Consider adding error handling for the `getUsers` function to manage potential errors gracefully.